src/components/molecules/InboxItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div :class="[selected ? 'bg-blue-100':'', 'flex items-center w-full h-16 md:h-20 border-2 border-transparent focus:border-black hover:bg-gray-100']">
+    <div class="p-4">
+      <p class="bg-primary-light rounded-full flex items-center justify-center font-heading w-10 h-10 md:w-12 md:h-12 text-3xl">
+        J
```

Comment:
I recognize you said that the J is a placeholder. We don't know if that will be an image or a letter that is being passed a prop. 
Regardless, something will be passed down so maybe we can add that as a prop as well? Maybe just the letter for now.

Reply:
I set it a prop type string. When an icon gets made I hope this will be split out into an atom component instead of roughed in